pkg/sheets/sheets.go

```diff
@@ -1,66 +1,154 @@
 package sheets
 
 import (
+	"bytes"
+	"context"
+	"crypto/md5"
+	"encoding/json"
 	"fmt"
+	"github.com/virtual-vgo/vvgo/pkg/log"
 	"github.com/virtual-vgo/vvgo/pkg/storage"
-	"net/http"
-	"strconv"
+	"time"
 )
 
-const BucketName = "sheets"
+const DataFile = "__data.json"
```

Comment:
What happens when we have concurrent writes?

Reply:
The server uses a lock in redis to prevent races.